packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -61,19 +61,5 @@ export const getCashierValidations = cashier_arr => {
     }, {});
 };
 
-export const excluded_notifications = isMobile()
-    ? ['contract_sold', 'switched_to_real']
-    : [
-          'you_are_offline',
-          'password_changed',
-          'switch_to_tick_chart',
-          'contract_sold',
-          'maintenance',
-          'bot_switch_account',
-          'new_version_available',
-          'svg_needs_poi_poa',
-          'svg_needs_poa',
-          'svg_needs_poi',
-          'svg_poi_expired',
-          'switched_to_real',
-      ];
```

Comment:
If you remove these strings, it will effect the toast notification and push notification which might need to be display for certain users. As i know that everything which includes `svg` need to be display for CR clients.

Reply:
Yes i agree. The `switched_to_real` is removed here. Therefore i believe that it wont show